go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/shirou/gopsutil v2.21.11+incompatible
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
```

Comment:
The security advisory said it would be better to bump to the version below to fix the cve:
```
require golang.org/x/net v0.7.0
```

Reply:
done